js/admin.js

```diff
@@ -935,6 +935,10 @@ $(document).ready(function()
     if ($('.kpi-container').length) {
         refresh_kpis();
     }
+
+    $('.kpi-refresh').on('click', '.refresh-kpis', function () {
```

Comment:
I had to add this code to legacy js for the legacy refresh code to pick up the new refresh button, otherwise I would have to create all the javascript for refreshing KPIs, which I want to avoid doing in this PR.